tasks.c

```diff
@@ -1545,7 +1548,11 @@ static void prvInitialiseNewTask( TaskFunction_t pxTaskCode,
     pxNewTCB->xTaskRunState = taskTASK_NOT_RUNNING;
 
     /* Is this an idle task? */
+    #if(configNUM_CORES > 1)
+    pxNewTCB->xIsIdle = ( pxTaskCode == prvIdleTask ) || (pxTaskCode == prvMinimalIdleTask);
```

Comment:
To comply with the coding standard this will have to be written out long hand, rather than all on one line.